src/Schema/OracleSchemaManager.php

```diff
@@ -520,7 +520,7 @@ protected function fetchTableOptionsByTable(string $databaseName, ?string $table
         foreach ($metadata as $table => $data) {
             $data = array_change_key_case($data, CASE_LOWER);
 
-            $tableOptions[$table] = [
+            $tableOptions[$this->getQuotedIdentifierName($table)] = [
```

Comment:
Could you try using `$this->_getPortableTableDefinition($data)` instead? This should produce the same result but would be more correct and consistent with other schema managers (see https://github.com/doctrine/dbal/pull/6753 for example).

Reply:
This works with `_getPortableTableDefinition` but I had to update the `fetchAllAssociativeIndexed` to `fetchAllAssociative`